portlet/exoadmin/src/main/java/org/exoplatform/organization/webui/component/UIAccountEditInputSet.java

```diff
@@ -147,15 +147,21 @@ public boolean save(OrganizationService service) throws Exception {
         Query query = new Query();
         String email = getUIStringInput("email").getValue();
         query.setEmail(email);
-        if (!oldEmail.equals(email) && service.getUserHandler().findUsersByQuery(query, UserStatus.ANY).getSize() > 0) {
+        if (((oldEmail != null && !oldEmail.equals(email)) || oldEmail == null) && service.getUserHandler().findUsersByQuery(query, UserStatus.ANY).getSize() > 0) {
             // Be sure it keep old value
-            user.setEmail(oldEmail);
-            query.setEmail(oldEmail);
+            if (oldEmail != null) {
+                user.setEmail(oldEmail);
+                query.setEmail(oldEmail);
+            }
```

Comment:
These lines seems useless. `user` and `query` are 2 local variables which are not used after these lines. We can remove them IMO. WDYT ?